apps/daimo-mobile/src/logic/chain.ts

```diff
@@ -115,6 +147,45 @@ export class ViemChain implements Chain {
       lastBlockTimestamp: status.l2.blockTimestamp,
     };
   }
+
+  subscribeTransfers(address: Address): () => void {
+    const onLogs = function (logs: TransferLog[]) {
+      console.log(`[CHAIN] got ${logs.length} transfers for ${address}`);
+      for (const log of logs) {
+        if (!log.args.from || !log.args.to || !log.args.value) {
+          console.warn(`[CHAIN] skipping invalid Tranfer log ${log}`);
+          continue;
+        }
+
+        const wei = 10 ** coin.decimals;
+        const amount = (Number(log.args.value) / wei).toFixed(2);
```

Comment:
nit I think [`formatUnits`](https://viem.sh/docs/utilities/formatUnits.html) can also do this

Reply:
formatUnits doesn't have display decimals (so might output `"42.690418"` for 6-decimal USDC)